yarn/plugin-checks/sources/checks-run.command.ts

```diff
@@ -20,10 +20,8 @@ class ChecksRunCommand extends BaseCommand {
         configuration,
       },
       async (report) => {
-        await Promise.allSettled([
-          this.runCheck(project.cwd, ['lint'], report),
-          this.runCheck(project.cwd, ['typecheck'], report),
-        ])
+          await this.runCheck(project.cwd, ['typecheck'], report),
+          await this.runCheck(project.cwd, ['lint'], report),
```

Comment:
```suggestion
          await this.runCheck(project.cwd, ['typecheck'], report)
          await this.runCheck(project.cwd, ['lint'], report)
```
Запятые не нужны же?